src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'f' ? 'husband' : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {age ? <p className="Person__age">I am {age}</p> : null}
+      {isMarried === true ? (
+        <p className="Person__partner">
+          {partnerName} is my {partner}
+        </p>
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
You also can use it inside element ( not critical )

Reply:
Hi Danil, thanks for advise)))